Library/Formula/h264enc.rb

```diff
@@ -0,0 +1,72 @@
+class H264enc < Formula
+  homepage "http://h264enc.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/h264enc/h264enc/h264enc-9.9.0.tar.gz"
+  sha1 "8bfa7017386a4b7612acd21e48dedf940652582d"
+
+   # mplayer needs x264 installed first 
+  depends_on "x264"
+  depends_on "mplayer"
+  depends_on "lsdvd"
+  depends_on "libbluray"
+  depends_on "flac"
+  depends_on "faac"
+  depends_on "lame"
+  depends_on "mkvtoolnix"
+  # >= 0.4.5
+  depends_on "gpac"
+  depends_on "ogmtools"
+  depends_on "vorbis-tools"
+  
+  def install
+
+    # Test mencoder, is there a better way?
+    ohai "Checking 'MEncoder' for x264 encoding"
+    begin
+	system "test ! -z \"$(mencoder -ovc help 2>/dev/null | awk '{print $1}' | grep 'x264')\""
+    rescue BuildError
+    	onoe "MEncoder wasn't built with x264. Uninstall 'brew uninstall mplayer' then reinstall 'brew install x264 mplayer'"
+	raise
+    end
+
+    bin.mkpath
+    man.mkpath
+    doc.mkpath
+
+    # fix the termcap issues using tput instead (could replace \e with \x1B or \033
+    inreplace "h264enc" do |s|
```

Comment:
Just for clarification, do you want me to patch mplayer.rb to include a "with-x264" option?

Along a similar line, does brew know if I say the following in my formula:

    depends_on "mplayer" => "with-x264"

That it knows mplayer was built with x264 and if not reinstall it?